packages/eslint-plugin-okta/lib/rules/no-missing-i18n-comments.test.js

```diff
@@ -0,0 +1,150 @@
+const rule = require('./no-missing-i18n-comments');
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('babel-eslint'),
+  parserOptions: {
+    sourceType: 'module',
+  },
+});
+
+function buildTestCode(codeLines) {
+  return `module.exports = {\n  ${codeLines.join('\n  ')}\n};`;
+}
+
+ruleTester.run('no-missing-i18n-comments', rule, {
+  valid: [
+    buildTestCode([
+      '// Description of single',
+      'key: "single"',
+    ]),
+    buildTestCode([
+      '// {0} refers to something',
+      'key: "{0}"',
+    ]),
+    buildTestCode(['key: "foo bar",']),
+    buildTestCode([
+      '// here is a long comment with a description for {0} and {1}',
+      'key: "{0} and {1}",',
+    ]),
+    buildTestCode([
+      '// First comment',
+      '// Second: {0} refers to something',
+      'key: "{0}",',
+    ]),
+    buildTestCode([
+      '// First comment with reference {0}',
+      '// Second does not',
+      'key: "{0}",',
+    ]),
+    buildTestCode([
+      '// Comment with reference {0}',
+      'key: "{0}",',
+      '// Another comment',
+      'key2: "Some data",',
+      '// Another param {0}',
+      'key3: "{0} - here",'
+    ]),
+    {
+      code: buildTestCode([
+        '// Comment explaining {{0}}',
+        'key: "{{0}}",'
+      ]),
+      options: [{ templateDelimiters: ['{{', '}}'] }],
```

Comment:
Correct - in `okta-ui-react`, arguments are surrounded in double curly braces instead of single.

This option allows us to remain flexible, supporting any syntax convention for positional arguments.

Reply:
Gotcha, I see that now. Thanks!